### PR TITLE
Copyright/legal pages: corrections, updates

### PR DESCRIPTION
**Pull request by grahamperrin**
<https://bugs.freebsd.org/270229#c10> (comment 10) onwards, 

> 270229 – FreeBSD Copyright and Legal Information: corrections, updates

English (en) pages first …

### CODE REVIEW

**Comment by bsdimp** [comment]
I'm not sure that this renaming makes sense. At least the names you pick are made up and don't follow any standards.
the files themselves appear to be one of the many lightly edited variations. SPDX has the canonical versions of the GPL and LGPL on its web site  https://spdx.org/licenses/LGPL-2.0-or-later.html and https://spdx.org/licenses/GPL-2.0-or-later.html so is there any reason we don't just link to them? 

But the bigger question is: Why not just delete them entirely? There's no reason to treat the GPL with any specialness anymore. We have almost no GPL code in the tree, and there's a dozen other licenses that we could link here. But it's not relevant for them either, I don't think, since this is just the project's added copyright terms (for the collection). We're not legally obligated to have these links in this location, so i'd just delete these files entirely and remove the links.

File: website/content/hu/copyright/_index.adoc [hunk 1]
```diff
@@ -5,16 +5,14 @@ sidenav: about
...
-== link:https://www.FreeBSD.org/copyright/license/[A BSD licenc]
+link:https://www.FreeBSD.org/copyright/license/[A BSD licenc]
 
-== link:https://www.FreeBSD.org/copyright/daemon/[A BSD démon]
+link:https://www.FreeBSD.org/copyright/daemon/[A BSD Démon]
 
-== link:https://www.FreeBSD.org/copyright/COPYING[GNU GENERAL PUBLIC LICENSE]
-
-== link:https://www.FreeBSD.org/copyright/COPYING.LIB[GNU LIBRARY GENERAL PUBLIC LICENSE]
+link:https://cgit.freebsd.org/ports/plain/COPYRIGHT[A FreeBSD Portokról]
```
Comment (grahamperrin) [inline website/content/hu/copyright/_index.adoc:18]: Google translation.

File: website/content/it/copyright/_index.adoc [hunk 2]
```diff
@@ -5,16 +5,14 @@ sidenav: about
...
-== link:https://www.FreeBSD.org/copyright/license/[Il Copyright di BSD]
+link:https://www.FreeBSD.org/copyright/license/[Il Copyright di BSD]
 
-== link:https://www.FreeBSD.org/copyright/daemon/[Il Demone BSD]
+link:https://www.FreeBSD.org/copyright/daemon/[Il Demone BSD]
 
-== link:https://www.FreeBSD.org/copyright/COPYING[GNU GENERAL PUBLIC LICENSE]
-
-== link:https://www.FreeBSD.org/copyright/COPYING.LIB[GNU LIBRARY GENERAL PUBLIC LICENSE]
+link:https://cgit.freebsd.org/ports/plain/COPYRIGHT[Port di FreeBSD]
```
Comment (grahamperrin) [inline website/content/it/copyright/_index.adoc:18]: Translation based on <https://www.freebsd.org/it/ports/>.

File: website/content/ja/copyright/_index.adoc [hunk 3]
```diff
@@ -5,18 +5,16 @@ sidenav: about
...
+link:trademarks[登録商標]
 
-== link:license[BSD の著作権]
+link:license[BSD の著作権]
 
-== link:daemon[BSD デーモン]
+link:daemon[BSD デーモン]
 
-== link:https://www.FreeBSD.org/copyright/COPYING[GNU GENERAL PUBLIC LICENSE]
+link:https://cgit.freebsd.org/ports/plain/COPYRIGHT[FreeBSD のポ]
```
Comment (grahamperrin) [inline website/content/ja/copyright/_index.adoc:18]: Partly from <https://www.freebsd.org/ja/ports/>, partly translated by Google.

File: website/content/ru/copyright/_index.adoc [hunk 4]
```diff
@@ -5,20 +5,18 @@ sidenav: about
...
-== link:https://www.FreeBSD.org/copyright/LEGAL[Ограничения на распространение портов FreeBSD]
+link:https://cgit.freebsd.org/ports/plain/COPYRIGHT[Портов FreeBSD]
 
-== link:https://www.FreeBSD.org/copyright/COPYING[Универсальная Общественная Лицензия GNU]
+link:https://www.FreeBSD.org/copyright/LEGAL[Ограничения на распространение портов FreeBSD]
 
-== link:https://www.FreeBSD.org/copyright/COPYING.LIB[Универсальная Общественная Лицензия GNU для библиотек]
-
-== link:https://www.FreeBSD.org/privacy/[Политика конфиденциальности FreeBSD]
+link:https://www.FreeBSD.org/privacy/[Политика конфиденциальности FreeBSD]
```
Comment (grahamperrin) [inline website/content/ru/copyright/_index.adoc:22]: Partly from <https://www.freebsd.org/ru/ports/>, partly translated by Google.

File: website/content/zh-cn/copyright/_index.adoc [hunk 5]
```diff
@@ -5,16 +5,14 @@ sidenav: about
...
-== link:license/[BSD 版权]
+link:license/[BSD 版权]
 
-== link:daemon/[BSD Daemon]
+link:daemon/[BSD Daemon]
 
-== https://www.FreeBSD.org/copyright/COPYING[GNU GENERAL PUBLIC LICENSE]
-
-== https://www.FreeBSD.org/copyright/COPYING.LIB[GNU LIBRARY GENERAL PUBLIC LICENSE]
+link:https://cgit.freebsd.org/ports/plain/COPYRIGHT[FreeBSD Ports]
```
Comment (grahamperrin) [inline website/content/zh-cn/copyright/_index.adoc:18]: Not translated to zh-cn.

File: website/content/ru/copyright/_index.adoc [hunk 6]
```diff
@@ -5,20 +5,18 @@ sidenav: about
...
+link:trademarks/[Информация о товарных знаках]
 
-== link:license/[Лицензия BSD]
+link:license/[Лицензия BSD]
 
-== link:daemon/[Даемон BSD]
+link:daemon/[Даемон BSD]
 
-== link:https://www.FreeBSD.org/copyright/LEGAL[Ограничения на распространение портов FreeBSD]
+link:https://cgit.freebsd.org/ports/plain/COPYRIGHT[Портов FreeBSD]
```
Comment (grahamperrin) [inline website/content/ru/copyright/_index.adoc:18]: Partly from <https://www.freebsd.org/ru/ports/>, partly translated by Google.

**Comment by grahamperrin** [comment]
<https://github.com/freebsd/freebsd-doc/pull/169#issuecomment-1508517496>: 

> I'm not sure that this renaming makes sense. …

True. I couldn't have two files both named `COPYING.txt`, so I named each file descriptively for draft purposes. 

Easier to delete, as you suggested. 

----

Take to Phabricator now?

File: website/content/ru/copyright/_index.adoc (hunk 6 above)
Comment (pauamma) [reply website/content/ru/copyright/_index.adoc:18]: My (very dim and over 40 years old, so *do* check with a native speaker) recollection of Russian grammar says "Портов" is a genitive plural where a nominative plural ("Порты" maybe?) is needed.

**Comment by sergio-carlavilla** [comment]
@grahamperrin make only the changes in the English version, the translations teams will handle the others

**Comment by grahamperrin** [comment]
> … make only the changes in the English version, …

Can we leave the line, and have the one and only word – _Ports_ – non-translated?

File: website/content/ru/copyright/_index.adoc (hunk 6 above)
Comment (grahamperrin) [reply website/content/ru/copyright/_index.adoc:18]: @pauamma thank you, does <https://bsd.to/cyQg/raw> help? Output from: 

`time rg Порты /usr/doc`

**Comment by sergio-carlavilla** [comment]
> > … make only the changes in the English version, …
> 
> Can we retain the one new line, in the translations, with the one and only word – _Ports_ – non-translated?

only Doceng members can make changes to all translations. the rest must be handled by each translation team. and I can tell you that from Spanish and Catalan (my mother tongues) you can't trust Google Translate and things like that... it makes it really bad

**Comment by grahamperrin** [comment]
I'm slightly confused, because the other three PRs covered multiple languages, and "English (en) pages first …" should have been a hint that this PR would, similarly, cover multiple languages. 

Can we, at least, go with the deletions, and the heading level changes?

File: website/content/ru/copyright/_index.adoc (hunk 6 above)
Comment (pauamma) [reply website/content/ru/copyright/_index.adoc:18]: > @pauamma thank you, does https://bsd.to/cyQg/raw help? Output from:
> 
> `time rg Порты /usr/doc`

It doesn't make me a native speaker, if that's what you're asking. But given what Carlavilla wrote, that's immaterial anyway.

**Pull request closed without merging**